odxtools/cli/_print_utils.py

```diff
@@ -109,8 +107,7 @@ def print_service_parameters(service: DiagService,
         print_fn(f" Negative Response '{resp.short_name}':")
         print_fn(f"   Parameters:\n")
         table = extract_parameter_tabulation_data(list(resp.parameters))
-        table_str = textwrap.indent(tabulate(table, headers='keys', tablefmt='presto'), "    ")
```

Comment:
are you sure we don't need the indent anymore?

Reply:
No you are right. It needs indentation.

Reply:
I have added indentation to the table.